OpenOversight/app/models.py

```diff
@@ -102,10 +102,6 @@ class Face(db.Model):
     id = db.Column(db.Integer, primary_key=True)
     officer_id = db.Column(db.Integer, db.ForeignKey('officers.id'))
     img_id = db.Column(db.Integer, db.ForeignKey('raw_images.id'))
-    face_position_x = db.Column(db.Integer, unique=False)
-    face_position_y = db.Column(db.Integer, unique=False)
-    face_width = db.Column(db.Integer, unique=False)
-    face_height = db.Column(db.Integer, unique=False)
```

Comment:
hm, so if we ever want to do e.g. #173, it's convenient to have these boxes in the database